src/iris/bin/sender.py

```diff
@@ -1477,6 +1477,16 @@ def init_sender(config):
                                   slaves=config['sender'].get('slaves', []))
 
 
+def reset_master_only_metrics():
```

Comment:
I think it would be enough to add these to the `default_sender_metrics` dict at the top of the file, as all of those get set to 0 with each iteration by metrics.emit. Looks like `new_incidents_cnt` is already there.